cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -7,7 +7,15 @@ class HttpCache
 
   # Language header and cookie are needed to separately cache language-specific pages.
   LANGUAGE_HEADER = %w(Accept-Language)
-  LANGUAGE_COOKIES = %w(language_ pm)
+
+  DEFAULT_COOKIES = [
+    # Language drop-down selection.
+    'language_',
+    # Page mode, for A/B experiments and feature-flag rollouts.
+    'pm',
```

Comment:
For what it's worth I won't be using this yet, because it's not per-user (right?), in case you want to reduce cache variations.  But no problem if you're happy to put it in.